jina/peapods/pods/__init__.py

```diff
@@ -20,7 +20,63 @@
 from ...helper import get_public_ip, get_internal_ip, random_identity
 
 
-class BasePod(ExitStack):
+class ExitFIFO(ExitStack):
+    """
+    ExitFIFO changes the exiting order of exitStack to turn it into FIFO.
+    """
+
+    def __exit__(self, *exc_details):
```

Comment:
Please add a docstring, telling where you C&P this code from and what exactly is the solemn change, that you did (e.g. the `popleft` thingy).

Reply:
Could u suggest change?

Reply:
No, because I don't know, where you copied the code from. And don't understand the consequences deeply enough.